stories/object-attribute/object-attribute.visual.js

```diff
@@ -0,0 +1,17 @@
+import * as stories from './object-attribute.stories.js';
```

Comment:
We just moved to automated visual testing via chromatic 🎉 , so you can remove this file
Docs: https://github.com/SAP/fundamental-styles/wiki/Visual-testing-with-Chromatic
PR: https://github.com/SAP/fundamental-styles/pull/1584